src/elements/common/annotator-context/withAnnotations.tsx

```diff
@@ -89,6 +119,26 @@ export default function withAnnotations<P extends object>(
             annotator.emit('annotations_remove', id);
         };
 
+        escalateAnnotationsFetchError = (error: ElementsError | Error) => {
+            const { onError } = this.props;
+
+            if (onError) {
+                const { message = '' } = error;
+                const origin = 'origin' in error && error.origin ? error.origin : ORIGIN_CONTENT_PREVIEW;
+                onError(
+                    {
+                        code: ERROR_CODE_FETCH_ANNOTATIONS,
+                        // eslint-disable-next-line @typescript-eslint/camelcase
+                        context_info: error,
+                        message,
+                        origin,
+                        type: 'error',
+                    },
```

Comment:
Could create a `createElementsError` or something like that to create the formatted error and avoid repeating it here and L198